jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>QC I agree is already part of the preprocessing chapter, but can we add a code snippet showing the cell type counts or have a function that filters all cell types with less than 50 cells. Many people will use this code just on a copy and paste basis so its nice to show why you are selecting those cell types.</p><p> </p><p>So essentially having the <code>value_counts</code> before the filtering.</p><p> </p><p><code>'None','Mixed_cells','Prol. cells','Lymphatic EC differentiating','Lymphatic EC proliferating' </code> you filter because of? Maybe add a sentence. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>OK we add "data.obs['cluster_labels_res.0.8'].value_counts()" before filtering. The cut-off x=50 cells for filtering out rare cells is a data specific parameter which depends on the cellular distribution. We can instead filter out cell types containing less that 5% of the overall population. </p>

Reply:
<p>Okay now?</p>

Reply:
<p>I don't understand why rare cell types need to be removed. Why not just keep them? I'd assume that a bulk deconvolution method would likely not find them in the bulk data, but that's okay?</p>

Reply:
<p>If these are very rare, you can't really trust the cell distribution you have to represent the 'true' cell type.</p><p>As someone that runs deconvolution a lot, I would not trust these cell types. It is better to remove them in this case.</p>

Reply:
<p>Is there a citation for this? If yes, please add your explanation and cite it! This is useful information, but it needs a citation.</p>